open-telemetry/Sources/App/Application+build.swift

```diff
@@ -127,7 +127,8 @@ func buildRouter() -> Router<AppRequestContext> {
     router.post("/wait") { request, _ in
         let time = try request.uri.queryParameters.require("time", as: Double.self)
         // Add child span
-        try await InstrumentationSystem.tracer.withSpan("sleep") { _ in
+        try await InstrumentationSystem.tracer.withSpan("sleep") { span in
```

Comment:
Probably a matter of taste, but you could use the top-level `withSpan` method instead of spelling out `InstrumentationSystem.tracer`.

Reply:
Changed, I didn't know about the top level function